src/Event/EventManager.php

```diff
@@ -22,7 +22,7 @@
  * can create multiple instances of this object to manage local events or keep a single instance
  * and pass it around to manage all events in your app.
  */
-class EventManager
+class EventManager implements EventManagerInterface
```

Comment:
The other methods in this class could have their doc blocks changed to be `inheritDoc` now.

Reply:
Yes, I’m not sure if we should add more methods to the interface. Other ones looked like internal/tests related.